types/pixi.js/index.d.ts

```diff
@@ -459,11 +459,15 @@ declare namespace PIXI {
             fn: (event: interaction.InteractionEvent) => void,
             context?: any
         ): this;
+        //tslint:disable-next-line:ban-types forbidden-types
+        on(event: string | symbol, fn: Function, context?: any): this;
         once(
             event: interaction.InteractionEventTypes,
             fn: (event: interaction.InteractionEvent) => void,
             context?: any
         ): this;
+        //tslint:disable-next-line:ban-types forbidden-types
+        once(event: string | symbol, fn: Function, context?: any): this;
```

Comment:
Instead of more forbidden-types, can you just change these to `(...args: any[]) => any`